net/DevExtreme.AspNet.Data/Compatibility.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace DevExtreme.AspNet.Data {
+
+    static class Compatibility {
+        public static IList UnwrapList(IList deserializedList) {
+            var unwrappedList = new List<object>();
+            foreach(var item in deserializedList)
+                unwrappedList.Add(UnwrapJsonElement(item));
+            return unwrappedList;
+        }
+
+        static object UnwrapJsonElement(object deserializeObject) {
+            if(!(deserializeObject is JsonElement jsonElement))
+                return null;
```

Comment:
Shouldn't an exception thrown here?

Reply:
It may be too aggressive against compatibility with existing scenarios. However, let's try to catch such cases via an exception. I do not mind. Will do.